src/components/AudioButton.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import { Howl } from 'howler'
+import { useEffect, useState, useRef } from 'react'
```

Comment:
Importing React is not necessary anymore with the newest "create-react-app" version.